subsys/bluetooth/controller/hci_internal.h

```diff
@@ -24,6 +27,45 @@
  */
 int hci_internal_cmd_put(uint8_t *cmd_in);
 
+/** A user implementable HCI command handler
+ *
+ * What is done in the command handler is up to the user.
+ * Parameters can be returned to the host through the raw_event_out output parameter.
+ *
+ * When the command handler returns, a Command Complete or Command Status event is generated.
+ *
+ * @param[in]  cmd               The HCI command itself. The first byte in the buffer corresponds
+ *                               to OpCode, as specified by the Bluetooth Core Specification.
+ * @param[out] raw_event_out     Parameters to be returned from the event as return parameters in
+ *                               the Command Complete event.
+ *                               Parameters can only be returned if the generated event is
+ *                               a Command Complete event.
+ * @param[out] param_length_out  Length of parameters to be returned.
+ * @param[out] gives_cmd_status  Set to true if the command is returning a Command Status event.
+ *
+ * @return Bluetooth status code. BT_HCI_ERR_UNKNOWN_CMD if unknown.
+ */
+typedef uint8_t (*hci_internal_user_cmd_handler_t)(uint8_t const * const cmd,
+						   uint8_t * const raw_event_out,
```

Comment:
```suggestion
typedef uint8_t (*hci_internal_user_cmd_handler_t)(uint8_t const * cmd,
						   uint8_t * raw_event_out,
```

Reply:
Will leave it as is if you're ok with it. Now it will match the existing code style in this file.

Reply:
Fixed